scripts/customer-info/customer_info.yaml

```diff
@@ -0,0 +1,224 @@
+    //typescript/...:
+    - '  darwin-fastbuild: 11'
+  'Execution Platforms:':
+    //cpp/...:
+    - '  @io_bazel_rules_dotnet//dotnet/toolchain:darwin_amd64_3.1.100: 8'
+    - ''
+    //csharp/...:
+    - '  @io_bazel_rules_dotnet//dotnet/toolchain:darwin_amd64_3.1.100: 196'
+    - ''
+    //docker/...:
+    - '  @io_bazel_rules_dotnet//dotnet/toolchain:darwin_amd64_3.1.100: 43'
+    - ''
+    //go/...:
+    - '  @io_bazel_rules_dotnet//dotnet/toolchain:darwin_amd64_3.1.100: 5'
+    - ''
+    //java/...:
+    - '  @io_bazel_rules_dotnet//dotnet/toolchain:darwin_amd64_3.1.100: 47'
+    //kotlin/...:
+    - '  @io_bazel_rules_dotnet//dotnet/toolchain:darwin_amd64_3.1.100: 8'
+    - ''
+    //scala/...:
+    - '  @io_bazel_rules_dotnet//dotnet/toolchain:darwin_amd64_3.1.100: 16'
+    - ''
+    //typescript/...:
+    - '  @io_bazel_rules_dotnet//dotnet/toolchain:darwin_amd64_3.1.100: 11'
+    - ''
+  'Mnemonics:':
+    //cpp/...:
+    - '  CcStrip: 1'
+    - '  SymlinkTree: 1'
+    - '  Middleman: 1'
+    - '  CppLink: 1'
+    - '  SourceSymlinkManifest: 1'
+    - '  CppCompile: 1'
+    - '  FileWrite: 2'
+    //csharp/...:
+    - '  SymlinkTree: 1'
+    - '  Middleman: 1'
+    - '  SourceSymlinkManifest: 1'
+    - '  FileWrite: 1'
+    - '  Compile: 1'
+    - '  CopyLauncher: 1'
+    - '  CopyFile: 190'
+    //docker/...:
+    - '  CcStrip: 1'
+    - '  TestRunner: 1'
+    - '  CppLink: 1'
+    - '  CppCompile: 1'
+    - '  Javac: 2'
+    - '  TemplateExpand: 2'
+    - '  JavaDeployJar: 2'
+    - '  FileWrite: 4'
+    - '  JavaSourceJar: 4'
+    - '  Fail: 4'
+    - '  Middleman: 7'
+    - '  SymlinkTree: 7'
+    - '  SourceSymlinkManifest: 7'
+    //go/...:
+    - '  SymlinkTree: 1'
+    - '  Middleman: 1'
+    - '  SourceSymlinkManifest: 1'
+    - '  GoLink: 1'
+    - '  GoCompilePkg: 1'
+    //java/...:
+    - '  GenProto: 1'
+    - '  TestRunner: 1'
+    - '  GenProtoDescriptorSet: 1'
+    - '  Action: 3'
+    - '  Middleman: 3'
+    - '  FileWrite: 3'
+    - '  TemplateExpand: 3'
+    - '  SymlinkTree: 3'
+    - '  SourceSymlinkManifest: 3'
+    - '  JavaDeployJar: 3'
+    - '  Turbine: 5'
+    - '  Javac: 8'
+    - '  JavaSourceJar: 10'
+    //kotlin/...:
+    - '  SymlinkTree: 1'
+    - '  Middleman: 1'
+    - '  SourceSymlinkManifest: 1'
+    - '  TemplateExpand: 1'
+    - '  KotlinFoldJarsRuntime: 1'
+    - '  JdepsMerge: 1'
+    - '  KotlinCompile: 1'
+    - '  JavaSourceJar: 1'
+    //scala/...:
+    - '  SymlinkTree: 1'
+    - '  Middleman: 1'
+    - '  TestRunner: 1'
+    - '  SourceSymlinkManifest: 1'
+    - '  TemplateExpand: 1'
+    - '  JavaIjar: 1'
+    - '  JavaSourceJar: 2'
+    - '  ScalaDeployJar: 2'
+    - '  Scalac: 2'
+    - '  FileWrite: 4'
+    //typescript/...:
+    - '  SymlinkTree: 1'
+    - '  Middleman: 1'
+    - '  TestRunner: 1'
+    - '  SourceSymlinkManifest: 1'
+    - '  FileWrite: 1'
+    - '  Genrule: 1'
+    - '  TsProject: 1'
+    - '  TsValidateOptions: 1'
+    - '  CopyFile: 3'
+  total_actions:
```

Comment:
There's not much context here and the same numbers seem to appear in platforms/configuration - maybe we can add up these numbers and present a total action count for the entire project?

Reply:
referring to the numbers under `total_actions`

Reply:
Done